internal/relationtuple/swagger_definitions.go

```diff
@@ -70,15 +70,15 @@ type getRelationsParams struct {
 
 // The basic ACL relation tuple
 //
-// swagger:parameters postCheck createRelationTuple
+// swagger:parameters postHasRelationship createRelationships
 type bodyRelationTuple struct {
 	// in: body
 	Payload ketoapi.RelationQuery
 }
 
 // The basic ACL relation tuple
 //
-// swagger:parameters getCheck deleteRelationTuples
+// swagger:parameters hasRelationship deleteRelationships
```

Comment:
Please follow the naming conventions we established for parameters: https://intranet.ory.sh/docs/dev/api-guidelines#routes

Reply:
I'm not sure what you mean here. We are not naming the parameters, we just declare which endpoints they are being used in.